In [32]:
import pandas as pd
import numpy as np
from scipy import stats

data=pd.read_csv(r'C:\siddharth\MCL\analytix\7. Basic Stats - Case Study-1\creditpromo.csv')
a=data.dollars[data['insert'] == 'Standard']
b=data.dollars[data['insert'] != 'Standard']

stats.ttest_ind(a,b,equal_var=False)

Ttest_indResult(statistic=-2.260422726464996, pvalue=0.024226348191648994)

H0:  mean1=mean2
H1:  mean1 != mean2
test: 2 sampled unpaired ttest
Ttest_indResult(statistic=-2.260422726464996, pvalue=0.024226348191648994)
pvalue<significant value(.05) >> reject H0
stat: -ve >> left tailed test >> mean1 > mean2
promo failed